### PR TITLE
migrate web_advanced_search_x2x to Odoo10

### PR DESCRIPTION
**Pull request by rami-wafaie**

### CODE REVIEW

**Comment by hbrunn** [comment]
you should follow the [guide](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0) to move commits from one version to the other, don't do this in one big commit

**Comment by rami-wafaie** [comment]
@hbrunn Thanks for the feedback, This guidlines are very helpfull (This is my first odoo 10 Migration contribution) should I close this MR and open a new one with the new guidlines?
The code changes for odoo 10 are not that big just some small adjustments to the js file and in the qweb template

**Comment by hbrunn** [comment]
just force push your new branch over this one, no need for a new PR

**Comment by rami-wafaie** [comment]
Done

File: web_advanced_search_x2x/static/src/js/web_advanced_search_x2x.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /* Copyright 2015 Therp BV <http://therp.nl>
  * Copyright 2017 Jairo Llopis <jairo.llopis@tecnativa.com>
+ * Copyright 2017 Rami Alwafaie <rami.alwafaie@initos.com>
```
Comment (hbrunn) [inline web_advanced_search_x2x/static/src/js/web_advanced_search_x2x.js]: I don't believe changing two lines merits a copyright claim

File: web_advanced_search_x2x/__manifest__.py [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
+# Copyright 2017 initOS GmbH - Rami Alwafaie
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
     "name": "Search x2x fields",
-    "version": "9.0.1.0.0",
+    "version": "10.0.1.0.0",
     "author": "Therp BV, "
               "Tecnativa, "
+              "initOS GmbH, "
```
Comment (hbrunn) [inline web_advanced_search_x2x/__manifest__.py]: you're a contributor, not an author

**Review by hbrunn — CHANGES_REQUESTED** [review]

**Review by hbrunn — APPROVED** [review]

**Review by felixvillafranca — APPROVED** [review]
Funtional review Ok.

**Review by yajo — APPROVED** [review]
Code OK

**Pull request merged**